packages/apollo-links/src/utils/createFile.js

```diff
@@ -96,10 +143,13 @@ const uploadFile = ({ file, createdFile }: UploadFileOptions): Promise<*> => {
 export const createFile = (
   { file, fileMeta }: UploadFileLinkOptions,
   mutate: (any) => any,
+  eventListeners?: XHREventListeners = {},
+  requestCallback?: (XMLHttpRequest) => any,
 ): Promise<CreatedFile> => new Promise((resolve, reject) => {
   const filename = getFileNameToUpload(file, fileMeta);
 
   mutate({
+    query: FILE_CREATE_MUTATION,
```

Comment:
Write a comment here for the next generation

Reply:
Done